### ISSUE TITLE
В базе Медучреждений есть объекты, не относящиеся к медицине

### ISSUE THREAD

**Issue by Yamortis** [Yamortis/Test_health.mail.ru#6]
**Предусловие:**
Т.к. аудит всех учреждений, находящихся в базе "Медучреждения", невозможен по причине отсутствия ресурсов, для иллюстрации наличия бага используется поиск по непрофильным запросам в разделе "Медучреждения" сервиса Здоровье.Мэйл.ру https://health.mail.ru/company/list_moscow/

**Шаги:**
1. В поисковой строке ввести немедицинский запрос (например: "миграция", "реклама").
2. В фильтре поиска справа выбрать раздел "в учреждениях".
3. Нажать кнопку поиска (лупу).

**Фактический результат:**
выдается список учреждений, не относящихся к медицине.
Скрин 1: https://yadi.sk/i/69zGzpoEzu4xzg
Скрин 2: https://yadi.sk/i/7n5LTjGwvQ9Xkw

**Ожидаемый результат:**
выдается список только медицинских организаций или ничего.

**Comment by Svetlana678** [comment]
Да, действительно ошибка. По содержанию все отлично, по оформлению ошибки есть следующие рекомендации:
Правопреемники: Никто не назначен - Назначить себя (так рекомендовалось в лекции)
Этикетки: Пока нет - Указать красную этикетку "ошибка".
